snippets/quick-order-product-row.liquid

```diff
@@ -0,0 +1,40 @@
+{% comment %}
+  Quick Order Product Row
+
+  Accepts:
+  - item: {Object} Variant or Product object
+  - image: {Object} Product or Variant image (optional)
+  Usage:
+  {% render 'quick-order-product-row' %}
+{% endcomment %}
+
+<tr
+  class="product"
+>
+  <td class="product-item__inner">
+    <div class="product-item__title--wrapper">
+      <div class="variant-item__media">
```

Comment:
Since it's a product row, unsure about the naming being `variant`. But we can change this later too 🤔